main.py

```diff
@@ -41,11 +11,19 @@ def test_pair_to_number(major_color, minor_color, expected_pair_number):
   pair_number = get_pair_number_from_color(major_color, minor_color)
   assert(pair_number == expected_pair_number)
 
+def print_color_code():
```

Comment:
Good try at segregating code 👍 
The file naming can still be improved to reflect the intent.
Also test code and production code (ref manual print) are in the same file.
It is a good idea to separate out test and production code - which is helpful in future extensions and maintenance